addons/l10n_cy/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+{
+    'name': 'Cyprus - Accounting',
+    'icon': '/account/static/description/l10n.png',
+    'countries': ['cy'],
+    'description': """
+Basic package for Cyprus that contains the chart of accounts, taxes, tax reports,...
+    """,
+    'license': 'LGPL-3',
+    'version': '1.0',
+    'category': 'Accounting/Localizations/Account Charts',
+    'depends': [
+        'account',
+        'base_vat',
```

Comment:
don't remember if that was really needed since we don't have written something for l10n_cy in base vat but since it's a backport i don't think we should remove it